tests/phpunit8.xml

```diff
@@ -0,0 +1,41 @@
+<phpunit
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/8.0/phpunit.xsd"
+    backupGlobals="false"
+    backupStaticAttributes="false"
+    colors="true"
+    convertErrorsToExceptions="true"
+    convertNoticesToExceptions="true"
+    convertWarningsToExceptions="true"
+    processIsolation="false"
+    stderr="true"
+    bootstrap="./phpunit.bootstrap.php"
+>
+    <php>
+        <env name="SYMFONY_DEPRECATIONS_HELPER" value="weak_vendors" />
+    </php>
+
+    <extensions>
+        <extension class="DAMA\DoctrineTestBundle\PHPUnit\PHPUnitExtension" />
+    </extensions>
+
+    <testsuites>
+        <testsuite name="DAMADoctrineTestBundle test suite">
+            <directory>../tests</directory>
+        </testsuite>
+    </testsuites>
+
+    <filter>
```

Comment:
is this part even needed (same for `phpunit.xml`)? wondering what it does :smile:

Reply:
It's for coverage, I simplified it.